18_conclusion/index.tex

```diff
@@ -1 +1,19 @@
-\chapter{Conclusion}
\ No newline at end of file
+\chapter{Conclusion}
+
+
+\begin{flushleft}
+    \begin{quote}
+        ``We are each our own devil, and we make this world our hell.''
+        \linebreak
+        \emph{Oscar Wilde}
+    \end{quote}
+\end{flushleft}
+
+Presented here, was the collective effort of 43 humble students united by a fascination with self-governance, Nash equilibria and a willingness to forgo their Christmas break. 
+
+This project successfully produced a simulation of a group of agents with the tools required to collaborate to solve collective action problems. Though limited in intelligence, in certain parameterisations, these agents were able to survive and thrive. This is no mean feat, especially considering that in the early days of this project, a frequently heard phrase was: ``How the fudge are we going to do this?''.
+
+Alongside the success of the simulation, we must also recognise the amazing accomplishment of meta self-organisation. The cohort spanned every timezone, personality, work ethic, haircut, political affiliation and ability to understand sarcasm (pardon the exaggerations here). Yet, despite all that divides us, we were able to come together and produce an impressive piece of work that we are particularly proud of. 
+
+I would like to conclude this report by thanking each and every one of my teammates for their outstanding contributions and unfailing committment to the delivery of this project. I think I speak for all of us when I say that I am amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please. \\
+- Preet Lalli
```

Comment:
```suggestion
We would like to conclude this report by thanking each and every one of our teammates for their outstanding contributions and unfailing commitment to the delivery of this project. We are all amazed at what we have created and it has truly been an experience like no other, but let's not do it again. Please. \\
```

Reply:
Seemed like Preet was thanking team 3, although that is not what she meant. I'd like to end by thanking everyone, cause honestly, you are all amazing

Reply:
im gonna change teammates to classmates. I think its fine for it to be in first person with preets name at the end. She did take on the responsibility for writing this part. And it helps the rest of the humour for it be designated to one human